texworks.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd /app/texlive/lib/perl5/site_perl/
+perl_version=$(ls)
+# add paths of TeXlive Flatpak extension binaries
+export PATH=/usr/bin:/app/bin:/app/texlive/bin:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux
```

Comment:
Since you are in a shell.
```suggestion
export PATH=/usr/bin:/app/bin:/app/texlive/bin:/app/texlive/bin/${FLATPAK_ARCH}-linux
```